packages/editor/src/plugins/text/components/suggestion-item.tsx

```diff
@@ -0,0 +1,70 @@
+import IconFallback from '@editor/editor-ui/assets/plugin-icons/icon-fallback.svg'
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import React, { forwardRef } from 'react'
+
+import { SuggestionOption } from '../hooks/use-suggestions'
+import { cn } from '@/helper/cn'
+
+interface SuggestionItemProps {
+  option: SuggestionOption
+  selected: boolean
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onFocus: () => void
+  onMouseEnter: () => void
+  onMouseLeave: () => void
+  onBlur: () => void
+  tooltipPosition?: 'right' | 'left'
+}
+
+export const SuggestionItem = forwardRef<
+  HTMLButtonElement,
+  SuggestionItemProps
+>((props, ref) => {
+  const {
+    option,
+    selected,
+    onClick,
+    onFocus,
+    onMouseEnter,
+    onMouseLeave,
+    onBlur,
+    tooltipPosition,
+  } = props
+
+  const { pluginType, title, icon, description } = option
+
+  const handleMouseEvents = {
+    onMouseEnter,
+    onMouseLeave,
+  }
```

Comment:
Minor: I don't like this, simply because it's more lines than just doing it inline.

Reply:
Removed